src/pybindings/xcdf/xcdf.cpp

```diff
@@ -0,0 +1,227 @@
+#include <sstream>
+#include <string>
+#include <cstdio>
+#include <limits>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/numpy.h>
+
+#include <xcdf/XCDF.h>
+#include "xcdf/XCDFFieldDescriptor.h"
+#include "xcdf/XCDFFile.h"
+
+namespace py = pybind11;
+
+std::string getVersion()
+{
+    std::stringstream ss;
+    ss << XCDF_MAJOR_VERSION
+       << "." << XCDF_MINOR_VERSION
+       << "." << XCDF_PATCH_VERSION;
+    return ss.str();
+}
+
+int write_test_file()
```

Comment:
I'd give this a path as argument: `write_test_file(const std::string& path)` and then use the `tmp_path` fixture to not create it in the current directory.

Reply:
It should also be `void`, there is no point to return 0 here, it is not a main function.